ros_gz_bridge/test/bridge_config.cpp

```diff
@@ -70,15 +70,16 @@ class BridgeConfig : public ::testing::Test
 
 TEST_F(BridgeConfig, Minimum)
 {
-  auto results = ros_gz_bridge::readFromYamlFile("test/config/minimum.yaml");
+  auto results = ros_gz_bridge::readFromYamlFile(
+    "package://ros_gz_bridge/test/config/minimum.yaml");
```

Comment:
Can we keep at least one test that uses the old file path (I.e, just `test/config/minimum.yaml`)  so we are sure we're not breaking behavior?

Reply:
Sure. Changed in [13f4008](https://github.com/gazebosim/ros_gz/pull/511/commits/13f40087d99b92c5bc4c8b157a951aad1efc0d9f).